### PR TITLE
Gumball levels

### PR DESCRIPTION
**Pull request by caleybrock**

### CODE REVIEW

File: dashboard/config/locales/data.en.yml [hunk 1]
```diff
@@ -328,16 +328,17 @@ en:
...
+        'studio_gumball_5': "The \"repeat forever\" block allows you to run code continuously.  Can you attach blocks to it to help Gumball warm up for his race by moving left and right repeatedly?"
+        'studio_gumball_6': "Can you help Anton announce the start of a race by saying \"Ready, set, go!\"? To test your program, don’t forget to click Anton after you click Run!"
+        'studio_gumball_7': "Darwin is in the flag gathering contest! Help him run around by attaching the \"move\" block to each of the events. When you’re done, click Run and try to collect all of the flags."
+        'studio_gumball_8': "We’ve provided most of the code that keeps Anton (actor 2) moving left and right, as well as the code to move Banana Joe (actor 1) around the screen.  Can you finish the code and add blocks to the \"when actor touches\" event to make it so that you both play a sound and score a point when Banana Joe and Anton collide?  Press Run to play the game."
```
Comment (caleybrock) [inline dashboard/config/locales/data.en.yml]: 2 of these levels are still a bit long, but alice shortened most of them

File: apps/i18n/studio/en_us.json [hunk 2]
```diff
@@ -378,6 +378,7 @@
   "repeatDo": "do",
   "repeatForeverTooltip": "Execute the actions in this block repeatedly while the story is running.",
   "spaceInvasion": "Space invasion!",
+  "foodFight": "Food fight!",
```
Comment (breville) [inline apps/i18n/studio/en_us.json]: should these (titles?) be grouped with similar text?  they seem a bit unusual right here..

File: apps/i18n/studio/en_us.json [hunk 3]
```diff
@@ -499,7 +500,7 @@
   "setSpriteDiego": "to a Diego image",
   "setSpriteScrat": "to a Scrat image",
   "setSpriteAnais": "to a Anais image",
-  "setSpriteAntony": "to a Antony image",
+  "setSpriteAnton": "to a Anton image",
```
Comment (breville) [inline apps/i18n/studio/en_us.json]: "an" gets set automatically, right?

File: apps/src/studio/levels.js [hunk 4]
```diff
@@ -766,10 +764,17 @@ levels.iceage_6 = utils.extend(levels.playlab_6, {
   firstSpriteIndex: 3, // diego
   goalOverride: {} // This prevents the override from original playlab from being used
 });
-levels.gumball_6 = utils.extend(levels.playlab_6, {
+
+levels.gumball_6 = utils.extend(levels.playlab_5, {
```
Comment (breville) [inline apps/src/studio/levels.js]: I'm a big fan of naming levels instead of using numbers, and now that we have this odd numbering (6 inheriting from 5) it seems like a good time to ponder that again.

File: apps/src/studio/levels.js [hunk 5]
```diff
@@ -1431,6 +1440,48 @@ levels.gumball_9 = utils.extend(levels.playlab_9, {
...
+  scale: {
+    snapRadius: 2
+  },
+  defaultEmotion: Emotions.NORMAL,
+  avatarList: ['bananajoe', 'anton'],
+  map: [
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [16,0, 0, 0, 0, 0,16, 0],
```
Comment (breville) [inline apps/src/studio/levels.js]: different comma alignment :)

File: apps/src/studio/levels.js [hunk 6]
```diff
@@ -1431,6 +1440,48 @@ levels.gumball_9 = utils.extend(levels.playlab_9, {
...
+      blockOfType('studio_throw', {SPRITE: 0, DIR: 2}) +
+      blockOfType('studio_playSound') +
+      blockOfType('studio_saySprite')
+    ),
+  startBlocks:
+    '<block type="when_run" deletable="false" x="20" y="20"></block>',
+  'progressConditions' : [
+    { required: { 'setEmotion': true, 'throwProjectile': true},
+      result: { success: true} },
+    { required: { 'timedOut': true}, result: {success: false}}
```
Comment (breville) [inline apps/src/studio/levels.js:1481]: cool

File: dashboard/config/locales/data.en.yml (hunk 1 above)
Comment (breville) [inline dashboard/config/locales/data.en.yml]: yeah, last one is still very long

File: dashboard/config/locales/data.en.yml [hunk 7]
```diff
@@ -328,16 +328,17 @@ en:
...
+        'studio_gumball_9': "Darwin just escaped a food fight! Change the background to \"characters\" and set Darwin’s speed to fast, then press Run and move him to Penny to warn her."
+        'studio_gumball_10': "Join the food fight! Use the \"actor __ throw __ right\" command to have Banana Joe (actor 1) throw food at Anton (actor 2)! Set Anton’s mood to angry when he gets hit."
```
Comment (breville) [inline dashboard/config/locales/data.en.yml]: i found the ___s hard to parse

**Comment by breville** [comment]
lgtm, just some small comments.  would love to see the levels named instead of numbered.

**Comment by caleybrock** [comment]
Renamed levels and fixed all comments except that one level text being too long.
